src/main/drivers/persistent.c

```diff
@@ -47,6 +47,17 @@ void persistentObjectWrite(persistentObjectId_e id, uint32_t value)
     RTC_HandleTypeDef rtcHandle = { .Instance = RTC };
 
     HAL_RTCEx_BKUPWrite(&rtcHandle, id, value);
+
+#ifdef SPRACING_PERSISTENT_RTC_WORKAROUND
```

Comment:
The naming for conditionals in Betaflight is `USE_...`.

Reply:
Done